examples/api/test/painting/star_border/star_border.0_test.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'package:flutter_api_samples/painting/star_border/star_border.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Finder getStartBorderFinder(StarBorder shape) {
+    return find.byWidgetPredicate(
+      (Widget widget) => widget is example.ExampleBorder && widget.border == shape,
+    );
+  }
+
+  testWidgets('Smoke test', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.StarBorderApp(),
+    );
+
+    expect(find.widgetWithText(AppBar, 'StarBorder Example'), findsOne);
+    expect(find.text('Star'), findsOne);
+    expect(find.text('Polygon'), findsOne);
+    expect(find.byType(SelectableText), findsExactly(2));
+    expect(find.byType(example.ExampleBorder), findsExactly(2));
+
+    expect(find.byType(Slider), findsExactly(6));
+    expect(find.text('Point Rounding'), findsOne);
+    expect(find.text('Valley Rounding'), findsOne);
+    expect(find.text('Squash'), findsOne);
+    expect(find.text('Rotation'), findsOne);
+    expect(find.text('Points'), findsOne);
+    expect(find.widgetWithText(OutlinedButton, 'Nearest'), findsOne);
+    expect(find.text('Inner Radius'), findsOne);
+    expect(find.widgetWithText(ElevatedButton, 'Reset'), findsOne);
+  });
+
+
+  testWidgets('StartBorder uses the values from the sliders', (WidgetTester tester) async {
```

Comment:
```suggestion
  });

  testWidgets('StartBorder uses the values from the sliders', (WidgetTester tester) async {
```

Reply:
Oops, sorry, here is the code updated [format: Remove spaces and update test description](https://github.com/flutter/flutter/pull/157401/commits/67beacb5cf2d07fde9f007326f55445d5cfbe133)